tests/format/js/conditional/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -131,127 +131,133 @@ c */? foo : bar : bar;
 
 =====================================output=====================================
 var inspect =
-  4 === util.inspect.length
-    ? // node <= 0.8.x
-      function (v, colors) {
-        return util.inspect(v, void 0, void 0, colors);
-      }
-    : // node > 0.8.x
-      function (v, colors) {
-        return util.inspect(v, { colors: colors });
-      };
+  4 === util.inspect.length ?
+    // node <= 0.8.x
+    function (v, colors) {
+      return util.inspect(v, void 0, void 0, colors);
+    }
+  : // node > 0.8.x
+    function (v, colors) {
+      return util.inspect(v, { colors: colors });
+    };
 
 var inspect =
-  4 === util.inspect.length
-    ? // node <= 0.8.x
-      function (v, colors) {
-        return util.inspect(v, void 0, void 0, colors);
-      }
-    : // node > 0.8.x
-      function (v, colors) {
-        return util.inspect(v, { colors: colors });
-      };
+  4 === util.inspect.length ?
+    // node <= 0.8.x
+    function (v, colors) {
+      return util.inspect(v, void 0, void 0, colors);
+    }
+  : // node > 0.8.x
+    function (v, colors) {
+      return util.inspect(v, { colors: colors });
+    };
 
-const extractTextPluginOptions = shouldUseRelativeAssetPaths
-  ? // Making sure that the publicPath goes back to to build folder.
+const extractTextPluginOptions =
+  shouldUseRelativeAssetPaths ?
+    // Making sure that the publicPath goes back to to build folder.
     { publicPath: Array(cssFilename.split("/").length).join("../") }
   : {};
 
-const extractTextPluginOptions2 = shouldUseRelativeAssetPaths
-  ? // Making sure that the publicPath goes back to to build folder.
+const extractTextPluginOptions2 =
+  shouldUseRelativeAssetPaths ?
+    // Making sure that the publicPath goes back to to build folder.
     { publicPath: Array(cssFilename.split("/").length).join("../") }
   : {};
 
-const extractTextPluginOptions3 = shouldUseRelativeAssetPaths // Making sure that the publicPath goes back to to build folder.
-  ? { publicPath: Array(cssFilename.split("/").length).join("../") }
+const extractTextPluginOptions3 =
+  (
+    shouldUseRelativeAssetPaths // Making sure that the publicPath goes back to to build folder.
+  ) ?
```

Comment:
This one is unfortunate. We can probably tweak the comment printer to output 

```js
shouldUseRelativeAssetPaths ? // Making sure that the publicPath goes back to to build folder.
```

without the need for parenthesis.

Reply:
any tips on how? I have a pretty hard time with comments in prettier, not very familiar with it